src/covvfit/plotting/_grid.py

```diff
@@ -58,3 +60,119 @@ def plot_grid(
                 over_diag(ax, i, j)
 
     return fig, axes
+
+
+@dataclass(frozen=False)
```

Comment:
In the notebook we are fortunate to use eight cities arranged in two columns.

I wanted to create a utility which automatically arranges a list of plots (one per city) in a grid and has convenient utilities to iterate over them.

For example, if we had 9 cities, they would be arranged in a 5 x 2 grid with 9 active axes and one inactive (and invisible) axis.

Reply:
super useful !